utils/author_catalog.py

```diff
@@ -64,6 +77,29 @@ def create_catalog(filepath: str = catalog_file_path) -> Dict:
     for row in read_csv(filepath):
         book = parse_data_row(row)
         if is_selected_author(book):
+            if log:
+                create_log(book)
+            if ";" in book.author:
+                book.author = book.author.split(";")[0]
+                # if author_book_catalog[book.author]:
+                #     for entry in author_book_catalog[book.author]:
+                #         if not book.title == entry.title:
+                #             author_book_catalog[book.author].append(book)
             author_book_catalog[book.author].append(book)
 
     return author_book_catalog
+
+
+def create_log(book: Book, log_path=LOG_FILE_PATH):
```

Comment:
Cool. Can you change the method name? It is not a log file. Sorry, it is my fault to say that way. Find a better name that captures the nature of the doc we are creating here.